### PR TITLE
fix for windows

### PR DESCRIPTION
**Pull request by memsharded**

### CODE REVIEW

File: 1pr_build_requires.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 from conans.test.utils.tools import TestClient
 from conans import tools
 
-me = os.path.abspath(os.path.dirname(__file__))
+me = os.path.abspath(os.path.dirname(__file__)).replace("\\", "/")
```
Comment (jgsogo) [inline 1pr_build_requires.py:7]: why? is it needed?

Comment (memsharded) [reply 1pr_build_requires.py:7]: Yes, otherwise in Windows fails.

Comment (jgsogo) [reply 1pr_build_requires.py:7]: why? how?

`conan export C:\\...\\path/recipe/conanfile.py` fails?

Comment (memsharded) [reply 1pr_build_requires.py:7]: No, not sure where it is, somewhere inside ``TestClient`` I guess, didn't investigate further.

Comment (memsharded) [reply 1pr_build_requires.py:7]: You have also removed the ``working_dir`` management? It is also a bit annoying that it fails and you need to create it manually.

Comment (jgsogo) [reply 1pr_build_requires.py:7]: It should work, the TestClient creates that folder if it doesn't exist

**Pull request closed without merging**